.travis.yml

```diff
@@ -16,8 +16,7 @@ before_script:
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
 - jdk_switcher use oraclejdk8
-- yarn rollup
 - yarn start > /dev/null 2>&1 &
-- sleep 5
+- sleep 10
```

Comment:
`yarn start` takes a little longer because we added rollup execution to the process